test/parallel_api/algorithm/alg.sorting/sort.pass.cpp

```diff
@@ -309,6 +309,7 @@ struct test_sort_op
 };
 
 #if TEST_DPCPP_BACKEND_PRESENT
+#    if __SYCL_UNNAMED_LAMBDA__
```

Comment:
another way to fix - https://github.com/oneapi-src/oneDPL/pull/664

Reply:
Please see my response in https://github.com/oneapi-src/oneDPL/pull/664 .